scss/os/overrides/_openlayers.scss

```diff
@@ -125,3 +125,11 @@
     color: #000;
   }
 }
+
+.ol-attribution li:not(:first-child):before {
+  content: ", "
```

Comment:
The Source/Sources label is the first child, so this is adding an extra comma before the first source.